pkgs/os-specific/linux/displaylink/default.nix

```
@@ -67,6 +67,7 @@ in stdenv.mkDerivation rec {
 
   meta = with stdenv.lib; {
     description = "DisplayLink DL-5xxx, DL-41xx and DL-3x00 Driver for Linux";
+    maintainers = with lib.maintainers; [ nshalman eyJhb abbradar peterhoeg ];
```

need to add yourself as a maintainer see #72738 for an example
```suggestion
    maintainers = with lib.maintainers; [ nshalman eyJhb abbradar peterhoeg ];
```

---

It's someone else who isn't a maintainer. I'll remove them. Thanks for the review. Changes incoming.

---

ah, interesting. sounds good